src/services/userService.js

```diff
@@ -0,0 +1,34 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const newMessageService = (newMessage, user, emiter) => {
+  const uniqueId = uuidv4();
+
+  if (!newMessage) {
+    return 'Message required';
+  }
+
+  if (!user) {
+    return 'User required';
+  }
```

Comment:
The validation for `user` returns a string. Consider returning an error object or throwing an error for consistency with other parts of the application.